src/main/java/blusunrize/immersiveengineering/common/blocks/metal/MixerTileEntity.java

```diff
@@ -172,17 +170,14 @@ public void tick()
 					if(!outputAll)
 					{
 						FluidStack inTank = this.tank.getFluid();
-						if(inTank!=null)
+						FluidStack out = Utils.copyFluidStackWithAmount(inTank, Math.min(inTank.getAmount(), 80), false);
+						int accepted = output.fill(out, FluidAction.SIMULATE);
+						if(accepted > 0)
 						{
-							FluidStack out = Utils.copyFluidStackWithAmount(inTank, Math.min(inTank.getAmount(), 80), false);
-							int accepted = output.fill(out, FluidAction.SIMULATE);
-							if(accepted > 0)
-							{
-								int drained = output.fill(Utils.copyFluidStackWithAmount(out, Math.min(out.getAmount(), accepted), false),
-										FluidAction.EXECUTE);
-								this.tank.drain(drained, FluidAction.EXECUTE);
-								ret = true;
-							}
+							int drained = output.fill(Utils.copyFluidStackWithAmount(out, Math.min(out.getAmount(), accepted), false),
+									FluidAction.EXECUTE);
+							this.tank.drain(drained, FluidAction.EXECUTE);
+							ret = true;
```

Comment:
It's a bit awkward to have this in the PR, but filtering it out now would be a pain too, so I guess we'll keep it

Reply:
That's what I get for telling IntelliJ to resolve all "constant boolean expression" problems in a file...

Reply:
And that's why I don't ever let IntelliJ auto-fix anything except imports. Because I had "rearrange entries" on once, and that was a mess.

Reply:
How else would you have dealt with simplifying all of that bounding box code? My approach was to make `fl`/`fw` final and set it to `Direction.NORTH`/`EAST`, and then let IntelliJ clean up all the expressions that had become constant. I don't think there was an option to do that just for one method.